test/client/views/requestDetails.js

```diff
@@ -20,13 +37,31 @@ describe('RequestDetailsView', function () {
 
         context('with a request that has an error', function () {
 
-            it('displays that the request has an error');
+            it('displays that the request has an error', function () {
+
+                var hasErrorStub = sinon.stub(this.model, 'hasError');
+
+                hasErrorStub.returns(true);
+
```

Comment:
Yep, the stub looks good. Moving it out of the before block was also a good move, keeping it local and executed only once if it's not needed for another test.

You can shorten this:
```
var hasErrorStub = sinon.stub(this.model, 'hasError');
hasErrorStub.returns(true);
```

to this:
```
sinon.stub(this.model, 'hasError').returns(true);
```